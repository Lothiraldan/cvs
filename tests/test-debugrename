#!/bin/sh

hg init
echo a > a
hg ci -Am t

hg mv a b
hg ci -Am t1
hg debugrename b

hg mv b a
hg ci -Am t2
hg debugrename a

echo % test with --rev
hg debugrename --rev 1 b

