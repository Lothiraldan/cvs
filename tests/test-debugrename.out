adding a
b renamed from a:b789fdd96dc2f3bd229c1dd8eedf0fc60e2b68e3
a renamed from b:37d9b5d994eab34eda9c16b195ace52c7b129980
% test with --rev
b renamed from a:b789fdd96dc2f3bd229c1dd8eedf0fc60e2b68e3
